next-env.d.ts

```diff
@@ -1,2 +1,7 @@
-/// <reference types="next" />
-/// <reference types="next/types/global" />
+/*
+ esse arquivo insere a tipagem padrão do next para a integração
+ com o typescript tanto para a utilização padrão das tipagens quanto
+ para plugins
```

Comment:
```suggestion
 para plugins.
```

Além disso, pode, por favor, colocar aqui a URL que eu pedi (a que fala desse arquivo no nextjs).